Tests/Editor/Extensions/TransformExtenstionsTest.cs

```diff
@@ -0,0 +1,62 @@
+﻿using NUnit.Framework;
+using StansAssets.Foundation.Extensions;
+using UnityEngine;
+
+namespace Tests.Editor.Extensions
+{
+    public class TransformExtenstionsTest
+    {
+        [Test]
+        [TestCase(0, false, false,ExpectedResult = 0)]
```

Comment:
Please, consider adding test names. So we can see great names in the _Test Runner_ window.
```suggestion
        [TestCase(0, false, false,ExpectedResult = 0, TestName = "Some good name")]
```

Reply:
Didn't noticed that option with [TestCase]. Thanks